### PR TITLE
AHTI-331 | Fix PriceTags not available in the API

### PR DESCRIPTION
**Pull request by charn**
## Description :sparkles:
This PR fixes an issue noticed while documenting the API. PriceTag instances were not available from the API.

### Related :handshake:
**[AHTI-331](https://helsinkisolutionoffice.atlassian.net/browse/AHTI-331):** Technical documentation

### CODE REVIEW

File: features/schema.py [hunk 1]
```diff
@@ -104,6 +104,7 @@ class Teaser(DjangoObjectType):
 
     class Meta:
         model = models.FeatureTeaser
+        fields = ()  # Don't include any fields from the model automatically
```
Comment (igordavydsson) [inline features/schema.py:107]: oh, cool 🙂

File: features/schema.py [hunk 2]
```diff
@@ -385,6 +381,9 @@ def resolve_details(self: models.Feature, info, **kwargs):
         for detail in self.details.all():
             # Default dict resolver will resolve this for FeatureDetails
             details[detail.type.lower()] = detail
+
+        # PriceTags have a relation to Feature model, so we resolve it separately
+        details["price_list"] = self.price_tags.all()
```
Comment (igordavydsson) [inline features/schema.py:386]: what's the reason for putting `PriceTag`s in `FeatureDetails`?

Comment (charn) [reply features/schema.py:386]: I think that was the intended end goal in PR #63. Some refactoring was done on that PR which left the implementation in a bit unpolished state. :sweat_smile:

Comment (igordavydsson) [reply features/schema.py:386]: ok, fair enough. it mentions some slack discussion as the reason for this choice 😄 we'll honor the discussion

**Pull request merged**